gui/src/pages/Status.jsx

```diff
@@ -66,7 +66,10 @@ export function Status(props) {
                     {x}
                   </Button>
                 ))}
-                {(status.status === 'running' || status.status === 'user_input' || status.status === 'error') && (
+                {(status.status === 'running' ||
+                  status.status === 'user_input' ||
+                  status.status === 'error' ||
+                  status.status === 'recipe_unsupported') && (
```

Comment:
Not required for this PR, but probably we want to make these constants in the UI as well.